test/user.test.js

```diff
@@ -363,6 +363,88 @@ describe('User', function() {
     });
   });
 
+  describe('Password length validation', function() {
+    var pass72Char = new Array(73).join('a');
+    var pass73Char = new Array(74).join('a');
+    var passTooLong = pass72Char + 'WXYZ1234';
+
+    it('rejects passwords longer than 72 characters', function(done) {
+      try {
+        User.create({ email: 'b@c.com', password: pass73Char }, function(err) {
+          if (err) return done (err);
+          expect(err).to.have.property('message', /Password too long/);
+        });
+      } catch (e) {
+        done();
+      }
```

Comment:
1) IIRC, the password validation throws an error. As a result, calling `User.create` may throw an error. I agree that's weird and it would be better to fix it. However, I think it's ok to leave this out of scope of this patch.

2) The test is still weird though. The callback function is non-sense, it short-circuits if `err` is set and then asserts `err` has some properties. The `catch` block is wrong too, because it accepts any error as test passing, even `TypeError: undefined is not a function`.